core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -600,11 +577,8 @@ private static Sep31InfoResponse sep31InfoResponseFromAssetInfoList(List<AssetIn
         AssetResponse assetResponse = new AssetResponse();
```

Comment:
We can remove the following. They've been checked in AssetValidator 
 
```
        boolean isQuotesSupported = assetInfo.getSep31().isQuotesSupported();
        boolean isQuotesRequired = assetInfo.getSep31().isQuotesRequired();
        if (isQuotesRequired && !isQuotesSupported) {
          throw new SepValidationException(
              "if quotes_required is true, quotes_supported must also be true");
        }
```

Reply:
Good catch. Removed in two places.